open_city_profile/settings.py

```diff
@@ -57,6 +57,7 @@
     USE_X_FORWARDED_HOST=(bool, None),
     CSRF_TRUSTED_ORIGINS=(list, []),
     TEMPORARY_PROFILE_READ_ACCESS_TOKEN_VALIDITY_MINUTES=(int, 2 * 24 * 60),
+    GDPR_AUTH_CALLBACK_URL=("", None),
```

Comment:
Shouldn't this be `(str, None)` or perhaps `(str, "")`.

Reply:
It should. :sweat_smile:

Reply:
Okay, fixed!